earth_enterprise/src/SConstruct

```diff
@@ -367,6 +367,40 @@ installdirs = {
     'htdocs': InstServerDir('gehttpd/htdocs'),
 }
 
+def get_cc_version(cc_dir):
```

Comment:
This should be changed back to `cc_path`. This is the path to the compiler whose version is being tested; it's not a path to a directory.